src/software/ai/ai.cpp

```diff
@@ -4,9 +4,10 @@
 
 #include "software/ai/hl/stp/play/halt_play.h"
 #include "software/ai/hl/stp/stp.h"
+#include "software/ai/navigator/path_planner/theta_star_path_planner.h"
 
 AI::AI()
-    : navigator(std::make_shared<Navigator>()),
+    : navigator(std::make_shared<Navigator>(std::make_unique<ThetaStarPathPlanner>())),
```

Comment:
Looks like the make_shared of make_unique is still here

Reply:
What should it be changed to?

Reply:
If possible, I'd make them both unique pointers, but otherwise I don't see a huge issue here?

Reply:
Ohhh sorry didn't realize the Navigator constructor needed the path planner like that now, and forgot `Navigator` was no longer an abstract type.

Reply:
the navigator may need to stay a shared_ptr right now for visualization

Reply:
Are we just exposing the navigator from the AI? :grimacing: That's kind of sub-ideal, generally we don't want to expose internal implementation details like that. Either way, this concern isn't relevant in this PR.

Reply:
ok I'll revert that then...

Reply:
Done

Reply:
@garethellis0 well the navigator used to be an interface...

The visualization concerns should be addressed once the interfaces are done. It was mostly a convenient way to get "public" navigator data at the time.